src/assets/config/config.dev.json

```diff
@@ -1,7 +1,6 @@
-{
-  "env": {
-    "name": "dev"
-  },
+{  "env": {
```

Comment:
use the version on the left  and just your keys "APIPreparersServerUri" &   "APITestStationsServerUri"